admin/services/class-indexable.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin\Services
+ */
+
+/**
+ * Represents the indexable service.
+ */
+class WPSEO_Indexable_Service {
+
+	/**
+	 * Retrieves an indexable.
+	 *
+	 * @param WP_REST_Request $request The request object.
+	 *
+	 * @return WP_REST_Response The response.
+	 */
+	public function get_indexable( WP_REST_Request $request ) {
+
+		$object_type = $request->get_param( 'object_type' );
+		$provider = $this->get_provider( $object_type );
```

Comment:
Let's make sure the `$object_type` is in lowercase when passing to the `get_provider` method.